### PR TITLE
Disable drag-and-drop when editing layer name.

### PR DESCRIPTION
**Pull request by shaoshing**
Fix for issue #3502

### CODE REVIEW

File: src/js/jsx/sections/layers/LayerFace.jsx [hunk 1]
```diff
@@ -153,6 +151,12 @@ define(function (require, exports, module) {
             this.getFlux().actions.groups.setGroupExpansion(currentDocument, currentLayer,
                 !currentLayer.expanded, descendants);
         },
+        
+        _handleLayerNameInput: function (focused) {
```
Comment (volfied) [inline src/js/jsx/sections/layers/LayerFace.jsx]: This requires JSDoc. I would also name it something more explicit about what it is doing. It's confusing, to me at least, to see a function name handleLayerNameInput which does nothing with the layer name being inputted.

**Comment by volfied** [comment]
One nit, looks great otherwise!

**Comment by shaoshing** [comment]
Thanks for reviewing. I added JSDoc and switched to a better name for the focus event handler (`_handleInputFocusChange`). Back to you @volfied

**Comment by volfied** [comment]
Let'er rip!

**Pull request merged**